duckduckgo/Android
# Unify styles used on buttons for all onboarding screens when concept test is enabled

Task/Issue URL: https://app.asana.com/0/0/1160191165900874/f
Tech Design URL: 
CC: https://app.asana.com/0/0/1160061676123057/f

**Description**:
For variants with the concept test enabled, show buttons with rounded corners and lowercase text in all onboarding screens. (This mainly applies to our Default browser CTA).

> Disclaimer: This is a short-living change, as we plan to redesign our default browser CTA. This change is only with the purpose of keeping UI consistency in the scope of this experiment.

**Steps to test this PR**:
1. Hardcode variant `mz`
1. Perform a fresh install
1. Navigate through the onboarding screens till Default Browser CTA is shown
1. Ensure buttons have rounded corners (as in the previous screen) and text is not uppercase.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
